src/vehicle.cpp

```diff
@@ -3044,18 +3049,18 @@ int vehicle::total_power(bool const fueled) const
     return pwr;
 }
 
-int vehicle::acceleration(bool const fueled) const
+float vehicle::acceleration(bool const fueled) const
 {
     if( ( engine_on && has_engine_type_not( fuel_type_muscle, true ) ) || skidding ) {
-        return safe_velocity( fueled ) * k_mass() / ( 1 + strain () ) / 10;
+        return safe_velocity( fueled ) * k_mass() / ( 1.0f + strain () ) / 10.0f;
```

Comment:
These changes are similar pointless: `strain` returns `float`, `1 + strain()` is a `float` as well, the multiplication will be done using floating point variables anyway.

Reply:
I prefer to change all numbers to the expected return type when possible. I find it improves code readability. When the types are mixed it forces the reader to spend time considering type/operator precedence to calculate the result. e.g.

int cls::fn() // Returns <10
...
float foo = fn / 10; // Always 0.
float foo = fn / 10.0f; // Functions as expected.